### ISSUE TITLE
Found unimported in same namespace 

### ISSUE THREAD

**Issue by piszczek** [sirbrillig/phpcs-import-detection#37]
```FILE: /opt/app/src/Domain/Entity/AssetUser.php
------------------------------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
------------------------------------------------------------------------------------------------
 8 | WARNING | Found unimported symbol 'Asset'.
   |         | (ImportDetection.Imports.RequireImports.Symbol)
------------------------------------------------------------------------------------------------
```

In this example `AssetUser.php` extends `AbstractAsset.php` which is in exacltly same namespace and same directory - so `use` statement isn't required here.

I tried to configure `ignoreGlobalsWhenInGlobalScope` to set `true` but it didn't help.

**Comment by sirbrillig** [comment]
Please see the discussion in #8 

TLDR: this sniff is just providing a scan of the current file and so it can't know what other files might have defined. It's safe to import something from the same namespace. If you'd prefer to scan multiple files, I suggest static analysis like [psalm](https://psalm.dev/) or [phpstan](https://github.com/phpstan/phpstan).

**Comment by piszczek** [comment]
ok, it makes sense, thx for reply 

__
edt. It would be good idea to mention about that in README (I searched for that before asking, I didn't check that same issue is closed)
